### ISSUE TITLE
RFE: support querying over HTTP

### ISSUE THREAD

**Issue by ktdreyer** [release-engineering/productmd#41]
User story: As a developer, I can query a compose over HTTP and retrieve a list of all builds in the compose.

Currently I can instantiate a Compose object like so:

```
    compose = productmd.compose.Compose(path)
    ... (and then query the list of builds, etc)
```

The problem is that "`path`" must be a local or NFS path, like `/mnt/redhat/path/to/my/compose`.

Within Red Hat, some of our NFS shares are restricted across data centers, and the composes are only accessible via HTTP. It would be great to improve productmd to support querying over HTTP instead of NFS.

**Comment by ktdreyer** [comment]
Here's how I did this with `requests` in the meantime.

```
LATEST_COMPOSE = 'http://url.example.com/path/to/latest-Ceph-2-RHEL-7/'
url = os.path.join(LATEST_COMPOSE, 'compose/metadata/rpm-manifest.json')
r = requests.get(url)
r.raise_for_status()
rpms = productmd.rpms.Rpms()
rpms.loads(r.text)
```

And now I can do something with the Rpms object, eg:

```
nvrs = set()
for arches in rpms.rpms.values():
    for builds in arches.values():
        for nevra in builds:
            nevra_dict = productmd.common.parse_nvra(nevra)
            nvrs.add('%(name)s-%(version)s-%(release)s' % nevra_dict)

# (... do something more with "nvrs" set, etc)

**Comment by dmach** [comment]
Fixed in 2d1f50df1f6746f690cd31b5c639341e4a47b852
